plutus-core/plutus-ir/test/TransformSpec.hs

```diff
@@ -209,6 +211,27 @@ inline =
             , "letOverAppMulti" -- multiple occurrences of an over-application of a function
             ]
 
+countLamTest :: TestNested
+countLamTest = testNested "countLamTest" $
+        map
+            (goldenPir (countLam . runQuote . PLC.rename) pTerm)
+            [ "var" -- from inline tests, testing let terms
```

Comment:
A bit too many tests for something as simple as `countLam`, since you've already written them it's fine to keep them.